doc/user/content/sql/create-source/postgres.md

```diff
@@ -27,9 +27,15 @@ Field | Use
 _src_name_  | The name for the source.
 **IF NOT EXISTS**  | Do nothing (except issuing a notice) if a source with the same name already exists. _Default._
 **CONNECTION** _connection_name_ | The name of the Postgres connection to use in the source. For details on creating connections, check the [`CREATE CONNECTION`](/sql/create-connection/#postgres) documentation page.
-**PUBLICATION** _publication_name_ | Postgres [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) (the replication data set containing the tables to be streamed to Materialize).
-**FOR ALL TABLES** | Creates subsources for all tables in the publication.
-**FOR TABLES** _table_name_ | Creates subsources for specific tables in the publication.
+**FOR ALL TABLES** | Create subsources for all tables in the publication.
+**FOR TABLES** _table_name_ | Create subsources for specific tables in the publication.
+
+### `CONNECTION` options
+
+Field                                | Value     | Description
+-------------------------------------|-----------|-------------------------------------
+`PUBLICATION`                        | `text`    | **Required.** The Postgres [publication](https://www.postgresql.org/docs/current/logical-replication-publication.html) (the replication data set containing the tables to be streamed to Materialize).
+`TEXT COLUMNS`                       | `text[]`  | Decode data as `text` for specific columns that contain PostgreSQL types that are unsupported in Materialize.
```

Comment:
The `Value` here should more rightly be written as "a list of names", i.e.:
- When the statement's roundtripped it doesn't print the array syntax using brackets, it instead prints the list using parens
- The input type isn't `text`, which requires being placed in commas, but are instead a series of period-delimited identifiers, which I usually just call "names"